.github/workflows/pull-request.yml

```
@@ -0,0 +1,32 @@
+name: PR actions
+'on':
+  pull_request:
+    branches: [master]
```

I think it's better to use the `types` instead of branches, so we can run this workflow even if we target other branches (or rename the `master` branch to something else, such as `main`)
```suggestion
     types:
    - opened
    - synchronize
```

---

Cool, didn't knew that!